.github/workflows/swift-toolchain.yml

```diff
@@ -755,78 +753,39 @@ jobs:
-          host_arch: ${{ inputs.build_arch }}
-          components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
-          arch: ${{ matrix.arch }}
-
-      - uses: seanmiddleditch/gha-setup-ninja@96bed6edff20d1dd61ecff9b75cc519d516e6401 # master
-        if: inputs.build_os == 'Darwin'
-
-      - name: Install Swift Toolchain (Mac)
-        if: inputs.build_os == 'Darwin'
+      - name: Install Swift Toolchain
         uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
         with:
           branch: ${{ env.WORKAROUND_MACOS_PINNED_BOOTSTRAP_TOOLCHAIN_BRANCH }}
           tag: ${{ env.WORKAROUND_MACOS_PINNED_BOOTSTRAP_TOOLCHAIN_TAG }}
 
       - name: Build early swift-driver
         run: |
-          swift build `
-                --configuration release `
-                --arch ${{ matrix.arch }} `
-                --package-path ${{ github.workspace }}/SourceCache/swift-driver `
-                --build-path ${{ github.workspace }}/BinaryCache/swift-driver `
-
-      - name: Create installable package
-        run: |
-          if ("${{ matrix.os }}" -eq "Darwin") {
-              $PlatformName = "macosx"
-              $ExeSuffix = ""
-              $Cpu = "${{ matrix.arch }}"
-              $SourceBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" "swift-driver" "${Cpu}-apple-macosx" "release"
-          } else {
-              $PlatformName = "windows"
-              $ExeSuffix = ".exe"
-              if ("${{ matrix.arch }}" -eq "ARM64") {
-                $Cpu = "aarch64"
-              } else {
-                $Cpu = "x86_64"
-              }
-              $SourceBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" "swift-driver" "${Cpu}-unknown-windows-msvc" "release"
-          }
+          $env:SWIFTCI_USE_LOCAL_DEPS=1
 
-          # The Swift compiler expects a specific folder name for the early swift-driver build.
-          # TODO: Make this configurable upstream so we don't have to hardcode it here.
-          $InstallFolderName = "earlyswiftdriver-${PlatformName}-${Cpu}"
-          $InstallBinDir = Join-Path "${{ github.workspace }}" "BinaryCache" $InstallFolderName "release" "bin"
-
-          # Create the target folder.
-          New-Item -ItemType Directory -Path $InstallBinDir -Force
-
-          # Copy binaries.
-          $SwiftBinaries = @("swift-driver", "swift-help")
-          foreach ($bin in $SwiftBinaries) {
-              $binName = "${bin}${ExeSuffix}"
-              $binPath = Join-Path $InstallBinDir $binName
-              Copy-Item -Path "${SourceBinDir}/${binName}" -Destination $binPath -Force
-          }
+          swift build `
+            --configuration release `
+            --package-path ${{ github.workspace }}/SourceCache/swift-driver `
+            --build-path ${{ github.workspace }}/BinaryCache/swift-driver `
+            -Xlinker "${env:SDKROOT}/usr/lib/swift/windows/x86_64/swiftCore.lib"
```

Comment:
This won't work on macOS and will need to be figured out.

Reply:
Ack, I'll fix it.